lib/omniauth/strategies/untappd.rb

```diff
@@ -56,7 +56,7 @@ def token_params
       def raw_info
         access_token.options[:mode] = :query
         access_token.options[:param_name] = :access_token
-        @raw_info ||= access_token.get('http://api.untappd.com/v4/user/info').parsed['response']['user']
+        @raw_info ||= access_token.get('https://api.untappd.com/v4/user/info').parsed['response']['user']
```

Comment:
Line is too long. [105/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.